options/locale/locale_en-US.ini

```
@@ -824,14 +824,18 @@ issues.desc = Organize bug reports, tasks and milestones.
 issues.new = New Issue
 issues.new.title_empty = Title cannot be empty
 issues.new.labels = Labels
+issues.new.add_labels_title = Apply labels
 issues.new.no_label = No Label
 issues.new.clear_labels = Clear labels
+issues.new.no_items = Nothing to show
```

Hmm... I don't like `Nothing to show` - I think it might clang as it's too verbose.

How about:

```suggestion
issues.new.no_items = None
```
Or 

```suggestion
issues.new.no_items = No items
```